frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.component.ts

```diff
@@ -129,49 +152,22 @@ export class WorkPackageRelationsAutocomplete implements AfterContentInit {
         .ensureLoaded(workPackage)
         .then(() => {
           this.onSelected.emit(workPackage);
-          this.ngSelectComponent.close();
+          this.ngSelectComponent.ngSelectInstance.close();
```

Comment:
The same as before, the close event is really basic, maybe we should provide it from the OpAutocompleterComponent which leads me to think that maybe OpAutocompleterComponent should extend NgSelectComponent. Let's think about it for version 2.